### ISSUE TITLE
[BUG] Plugin bunkernet failed on init()  due to missing file  /opt/bunkerweb/cache/ip.list

### ISSUE THREAD

**Issue by chrismade** [bunkerity/bunkerweb#354]
**Description**
In a default configuration (no custom changes made to variables.env ) the service does not (seem to) start due to missing file  /opt/bunkerweb/cache/ip.list
the folder exists and already contains a file called "instance.id" 
throwing error
`Plugin bunkernet failed on init()`

**How to reproduce**
In a linux system (debian buster) install bunkerweb - following the instructions - with "apt install bunkerweb" then either check the logfiles with `journalctl -xe --no-pager` or `systemctl status bunkerweb` to find an error complaining about the missing file and no further messages which indicates that the start of the service has stopped at this point.

IIRC I also noticed this issue in the docker version of bunkerweb

a manual creation of an empty file with e.g.
`touch  /opt/bunkerweb/cache/bunkernet/ip.list`
fixes the problem and a restart starts the service as expected -

However, the creation of that (empty) file should either go into the install procedure after "instance.id" had been created or non-existence of an empty file should be ignored - instead of stopping the start of the service without error in the logs

**Logs**

```
Nov 14 23:52:42 debian11w start.sh[530]: [2022-11-14 23:52:42] GENERATOR - ℹ️ - Checking arguments ...
Nov 14 23:52:42 debian11w start.sh[530]: [2022-11-14 23:52:42] GENERATOR - ℹ️ - Checking core plugins orders ...
Nov 14 23:52:42 debian11w start.sh[530]: [2022-11-14 23:52:42] GENERATOR - ℹ️ - Computing config ...
Nov 14 23:52:42 debian11w start.sh[530]: [2022-11-14 23:52:42] GENERATOR - ℹ️ - Removing old files ...
Nov 14 23:52:42 debian11w start.sh[530]: [2022-11-14 23:52:42] GENERATOR - ℹ️ - Rendering templates ...
Nov 14 23:52:43 debian11w start.sh[530]: [2022-11-14 23:52:43] GENERATOR - ℹ️ - Generator successfully executed !
Nov 14 23:52:43 debian11w start.sh[418]: [2022-11-14 23:52:43] ENTRYPOINT - ℹ️ - Run jobs scheduler ...
Nov 14 23:52:43 debian11w start.sh[418]: [2022-11-14 23:52:43] ENTRYPOINT - ℹ️ - Start BunkerWeb ...
Nov 14 23:52:43 debian11w start.sh[532]: [2022-11-14 23:52:43] SCHEDULER - ℹ️ - Executing job scheduler ...
Nov 14 23:52:44 debian11w start.sh[534]: nginx: [error] [INIT] Plugin bunkernet failed on init() : error while reading database : /opt/bunkerweb/cache/bunkernet/ip.list: No such file or directory
```

**Comment by chrismade** [comment]
excellent - yes, I confirm this 'error' is not blocking from starting bunkerweb (which I found out later when I switched to the container version of BW) - however "error" always looks bad in logfiles expecially if you are new to this software and don't know which errors are actually "warnings" you could ignore - and which errors lead to situations where the software is not working as expected

**Comment by TheophileDiot** [comment]
I see your point, but in that case it was indeed an error (with the plugin) because if the file is missing you wouldn't have the banned ips available to BunkerNet.

We can maybe however add a log to confirm that the connection to BunkerNet in the lua file (even if the py file confirms it)

**Comment by fl0ppy-d1sk** [comment]
Hey @chrismade,

It should be fixed in the last [v1.4.8 release](https://github.com/bunkerity/bunkerweb/releases/tag/v1.4.8).

Thanks for your report !
